scala/scala.bzl

```diff
@@ -384,3 +424,190 @@ def scala_repositories():
     url = "https://oss.sonatype.org/content/groups/public/org/scalatest/scalatest_2.11/2.2.6/scalatest_2.11-2.2.6.jar",
     sha256 = "f198967436a5e7a69cfd182902adcfbcb9f2e41b349e1a5c8881a2407f615962",
   )
+
+
+# TODO is it possible to share scala-library with the above?
+def scrooge():
```

Comment:
can we move the rest of this into a second file (and probably second PR)? Can we tackle just srcjar first?